docker-compose.yml

```diff
@@ -366,11 +366,10 @@ services:
                 ip route add default via 40.40.40.1 dev eth0.40 table 1001
                 sleep 5 && \
                 ping -c 3 6.6.6.6 -I 40.40.40.40 && ping -c 3 40.40.40.41 && \
-                ip link add link eth1 name eth1.50 type vlan id 50 && \
-                ip link set eth1.50 up && \
-                ip addr add 50.50.50.50/24 dev eth1.50 && \
+                ip addr add 50.50.50.50/24 dev eth1 && \
                 ip rule add from 50.50.50.50 lookup 1002
-                ip route add default via 50.50.50.1 dev eth1.50 table 1002
+                ip route add default via 50.50.50.1 dev eth1 table 1002
+                arp -s 50.50.50.1 aa:bb:cc:00:00:51
```

Comment:
We dont need this anymore, please refer my above reply.

Reply:
I am curious why this is not happening with the Trunk case and only access case. I mean how the bridges on the host are reacting when they see packets with vlan tags on them taking into account that those bridges have no vlan configuration ? Do they drop the packets or they forward the packet based on the MAC dest ignoring the vlan tag ? Because I am curious how the virtual env works coreclty for trunk cases and only have problems for access case.

Reply:
removed static ARP https://github.com/opiproject/opi-evpn-bridge/pull/156/commits/ef000a4d003a030cc33cf1cfc1ac831a44a16f0f

Reply:
@mardim91 I'll check and get back for your questions.

Reply:
Ty @venkatmahalingam